sklearn/ensemble/tests/test_bagging.py

```diff
@@ -449,7 +449,7 @@ def test_parallel_classification():
     decisions2 = ensemble.decision_function(X_test)
     assert_array_almost_equal(decisions1, decisions2)
 
-    ensemble = BaggingClassifier(SVC(),
```

Comment:
Just a note for future: There seems to be a runtime warning due to division by zero.